server/src/schemas/stats/boards.stats.schema.ts

```diff
@@ -0,0 +1,60 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+import { StatsUserTimestamp } from 'src/shared/interfaces/StatsUserTimestamp';
+import { Schema as MongooseSchema } from 'mongoose';
+
+@Schema()
+export class BoardsStats extends Document {
+  @Prop({
+    type: Types.ObjectId,
+    ref: 'Board',
+    required: true,
+  })
+  boardId: Types.ObjectId;
```

Comment:
i think `_id` is created by default and it has `ObjectId` type
If it's sth different and i dont understand or you prefer it that way its fine

Reply:
This is schema for BoardStats, the boardId is a reference to the board.